### ISSUE TITLE
Roary 

### ISSUE THREAD

**Issue by namdeka** [ubccr/software-layer#121]
https://sanger-pathogens.github.io/Roary/

**Comment by dsajdak** [comment]
@namdeka would you please email CCR Help with two sample gff3 files so we can test this install?  I tried my best to get some in the right format but this is not my area of expertise and I was unsuccessful.  I get this message:
`All input files have been excluded from analysis. Please check you have valid GFF files, with annotation and a FASTA sequence at the end. Better still, reannotate your FASTA file with PROKKA. `

Thank you

**Comment by dsajdak** [comment]
@namdeka we're still waiting to hear back from you so this software can be tested and then published to our software repository.

**Comment by namdeka** [comment]
Hello,

I am out of town currently. I will get this to you by 02/07.

Thank you.


Namrata Deka
Ph.D. Candidate | Armbruster Lab
Vice President | Micro Club 2023-24
Department of  Microbiology and Immunology
University at Buffalo
Buffalo, NY
Telephone: (716) 480-0739
Twitter: @Deka_Namrata22
________________________________

**Comment by dsajdak** [comment]
Hi @namdeka - thank you for providing us with a test set of files.  We have published roary to the ccrsoft/2023.01 software environment.  You can find it and it's dependencies by using 'module spider'

```
[vortex:~]$ module spider roary

----------------------------------------------------------------------------------------------------------------
  roary: roary/3.13.0
----------------------------------------------------------------------------------------------------------------
    Description:
      Rapid large-scale prokaryote pan genome analysis

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "roary/3.13.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      Rapid large-scale prokaryote pan genome analysis


      More information
      ================
       - Homepage: https://github.com/sanger-pathogens/Roary

```
